src/main/java/frc/robot/Robot.java

```diff
@@ -21,6 +24,12 @@
  * project.
  */
 public class Robot extends TimedRobot {
+  enum RobotType {aBot, bBot}
+
+  AnalogInput robotCheckAnalog = new AnalogInput(0);
```

Comment:
private

Reply:
Also, the literal 0 should be a constant from Constants